site/public/js/forum.js

```diff
@@ -2216,13 +2216,13 @@ function updateThread(e) {
         edit_post_id: $('#edit_post_id').val(),
         csrf_token: $('input[name="csrf_token"]').val(),
         title: $('input#title').val(),
-        thread_post_content: $('textarea#reply_box_').val(),
+        thread_post_content: $('textarea#reply_box_0').val(),
```

Comment:
The `.val()` is used to get the current value of selected element, which will return the corresponding value of `reply_box_` so manually inserting a digit should not be the solution.

Reply:
The bug was that '#reply_box' didn't select anything because each reply box has an int appended to their id to distinguish them. The box in the edit post form is always zero (EditPostForm.twig line 13), so the hardcoded number would have worked. I did just make the change to select the post box id and use that though.